samples/SkiaMultiplatformSample/build.gradle.kts

```diff
@@ -163,51 +167,22 @@ kotlin {
                 }
                 else -> throw GradleException("Host OS is not supported")
             }
-            val iosMain by getting {
+            val iosMain by creating {
                 dependsOn(darwinMain)
             }
-            val iosX64Main by getting {
-                dependsOn(iosMain)
+            val iosSourceSetName = when (host) {
+                "macos-x64" -> "iosX64Main"
+                "macos-arm64" -> "iosSimulatorArm64Main"
+                else -> throw GradleException("Host OS is not supported")
             }
-            val iosArm64Main by getting {
-                dependsOn(iosMain)
+            getByName(iosSourceSetName).dependsOn(iosMain)
+            if (LAUNCH_ON_REAL_DEVICE) {
+                getByName("iosArm64").dependsOn(iosMain)
             }
         }
     }
 }
 
-if (hostOs == "macos") {
-    project.tasks.register<Exec>("runIosSim") {
```

Comment:
What would be the way to run it on simulator after this task is removed? Only via xcode or q
Appcode?